include/video.h

```diff
@@ -188,18 +188,18 @@ enum class PixelFormat : uint8_t {
 	// FFmpeg Equivalent: AV_PIX_FMT_BGR24
 	BGR24_ByteArray = 24,
 
-	// 16.7M (32-bit) true colour; 8 bits per red/blue/green component;
-	// stored as packed uint32 data with highest 8 bits unused
+	// 16.7M (32-bit) true colour, 8 bits per red/blue/green component;
+	// stored as packed 32-bit data
```

Comment:
"16.7M (32-bit) true colour, 8 bits per red/blue/green component **plus a padding byte**; stored as packed 32-bit data"

Isn't that more accurate and how 32-bit true colour on actual hardware works?